meson_options.txt

```diff
@@ -1,3 +1,8 @@
+# features
+option('video-support', type: 'feature', value: 'enabled')
```

Comment:
Wouldn't it be better to change the default value to disable?

Reply:
Video support is opposite of no_video. I followed the past options, still it could remain for a discussion.

Reply:
video support should be kept enabled by default.